src/workloads/execution/locf.yml

```diff
@@ -0,0 +1,257 @@
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {_id: 1},
+            partitionBy: "$part1",
+            output: {
+                string: {
+                  $locf: "$string"}
+                  }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+    - OperationMetricsName: TestRandomTypeWithSinglePartition
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {_id: 1},
+            partitionBy: "$part1",
+            output: {
+                randomType: {
+                  $locf: "$randomType"}
+                  }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+    - OperationMetricsName: TestNumericWithMultiplePartitions
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {_id: 1},
+            partitionBy: {part1: "$part1", part2: "$part2"},
+            output: {
+                numeric: {
+                  $locf: "$numeric"}
+                  }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+    - OperationMetricsName: TestDateWithMultiplePartitions
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {_id: 1},
+            partitionBy: {part1: "$part1", part2: "$part2"},
+            output: {
+                date: {
+                  $locf: "$date"}
+                  }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+    - OperationMetricsName: TestStringWithMultiplePartitions
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {_id: 1},
+            partitionBy: {part1: "$part1", part2: "$part2"},
+            output: {
+                string: {
+                  $locf: "$string"}
+                  }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+    - OperationMetricsName: TestRandomTypeWithMultiplePartitions
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {_id: 1},
+            partitionBy: {part1: "$part1", part2: "$part2"},
+            output: {
+                randomType: {
+                  $locf: "$randomType"}
+                  }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq: 
+        - standalone
+        - replica
+        - replica-all-feature-flags
```

Comment:
Given that we're going to be removing the feature flag before this is committed, I don't think we need the all-feature-flag variant.

Reply:
done